app/views/mods_panel.py

```diff
@@ -7,7 +7,7 @@
 from pathlib import Path
 from shutil import copy2, copytree, rmtree
 from traceback import format_exc
-from typing import List, Optional
+from typing import List
```

Comment:
while you're at it, List from typing can be just replaced with list[]

Reply:
Yeah i was just lazy. There was only a few left that i see. I took care of them.